pyproject.toml

```diff
@@ -32,7 +32,7 @@ classifiers = [
         "Operating System :: POSIX :: Linux",
 ]
 dependencies = [
-   "howso-engine~=16.1",
+   "howso-engine~=22.1",
```

Comment:
I don't think these versions actually need to be updated in this way? If my understanding of `~=` is correct, `~=22.0` should encompass anything `>=22.0.0` and so we only need the requirements to be re-generated.